CONTRIBUTING.md

```diff
@@ -0,0 +1,237 @@
+# Welcome to HacktoberFest
+
+<div align="center">
+<img src="https://user-images.githubusercontent.com/52650290/192514718-be2d966f-6335-474a-8fbf-1704b87aefe6.jpg" alt="hacktoberfest"/>
+</div>
+  
+Hacktoberfest is for everyone. It is a month-long celebration from October 1st - 31st sponsored by Digital Ocean and GitHub to get people involved in [Open Source](https://github.com/open-source). Create your very first pull request to any public repository on GitHub and contribute to the open source developer community.
+
+If you haven't register yet so register now: https://hacktoberfest.com/
+
+
+<!-- omit in toc -->
+# Contributing to Simple Library Management
```

Comment:
“Simple Library Management”?

Reply:
Ohh i thought contributing file contains how to contribute, so I do copied the content , i will modify it